### PR TITLE
design_user_detail_screen

### PR DESCRIPTION
**Pull request by chuongmd-1771**
### Issue Link :link:

https://edu-redmine.sun-asterisk.vn/issues/10026
### Goals :soccer:

- design user detail screen

### CODE REVIEW

**Comment by chuongmd-1771** [comment]
ready

**Comment by chuongmd-1771** [comment]
ready

File: Petty/Extensions/UIView+.swift [hunk 1]
```diff
@@ -28,4 +28,39 @@ extension UIView {
         layer.cornerRadius = cornerRadius
         layer.shadowRadius = shadowRadius
     }
+    
+    func addTopBorderWithColor(color: UIColor, width: CGFloat) {
+        let border = CALayer()
+        border.backgroundColor = color.cgColor
+        border.frame = CGRect(x: 0, y: 0, width: self.frame.size.width, height: width)
+        self.layer.addSublayer(border)
```
Comment (tienpx3010) [inline Petty/Extensions/UIView+.swift:36]: Bỏ self tương tự

File: Petty/Extensions/UIViewController+.swift [hunk 2]
```diff
@@ -19,4 +21,5 @@ extension UIViewController {
     func dismissKeyboard() {
         view.endEditing(true)
     }
+
```
Comment (tienpx3010) [inline Petty/Extensions/UIViewController+.swift:24]: Thừa dòng

File: Petty/Scenes/Login/LoginViewController.swift [hunk 3]
```diff
@@ -27,9 +28,29 @@ final class LoginViewController: UIViewController {
     }
     
     @IBAction func loginPressed(_ sender: Any) {
+        SVProgressHUD.show()
+        Auth.auth().signIn(withEmail: emailTextField.text!, password: passTextField.text!) { [weak self] (user, error) in
+            if error != nil {
```
Comment (tienpx3010) [inline Petty/Scenes/Login/LoginViewController.swift:33]: guard let self để unwrap

File: Petty/Scenes/Login/LoginViewController.swift [hunk 4]
```diff
@@ -27,9 +28,29 @@ final class LoginViewController: UIViewController {
...
+                    }
+                }
+                self?.performSegue(withIdentifier: "gotoHome", sender: self)
+            }
+        }
     }
     
     @IBAction func registerPressed(_ sender: Any) {
+        Auth.auth().createUser(withEmail: emailTextField.text!, password: passTextField.text!) { [weak self] (user, error) in
+            guard self != nil else { return }
```
Comment (tienpx3010) [inline Petty/Scenes/Login/LoginViewController.swift:48]: guard let self để unwrap

File: Petty/Scenes/Common/Constant.swift [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+//  Petty
+//
+//  Created by chuongmd on 4/8/19.
+//  Copyright © 2019 Sun-Asterisk. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+enum Alert {
```
Comment (tienpx3010) [inline Petty/Scenes/Common/Constant.swift:12]: enum Constant {
          enum Alert

File: Petty/Scenes/Common/AlertView.swift [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+//
+//  Created by chuongmd on 4/8/19.
+//  Copyright © 2019 Sun-Asterisk. All rights reserved.
+//
+
+import UIKit
+
+final class AlertView: NSObject {
+    
+    class func showAlert(view: UIViewController, message: String, alertTitle: String, actionTitle: String) {
```
Comment (tienpx3010) [inline Petty/Scenes/Common/AlertView.swift:13]:
```suggestion
    static func showAlert(view: UIViewController, message: String, alertTitle: String, actionTitle: String) {
```

File: Petty/Extensions/UIView+.swift [hunk 7]
```diff
@@ -28,4 +28,39 @@ extension UIView {
...
+    }
+    
+    func addRightBorderWithColor(color: UIColor, width: CGFloat) {
```
Comment (tienpx3010) [inline Petty/Extensions/UIView+.swift:39]:
```suggestion
    func addRightBorderWith(color: UIColor, width: CGFloat) {
```

**Pull request closed without merging**